mod_reforged/hooks/entity/world/location.nut

```diff
@@ -5,4 +5,12 @@
 		onLeave();
 		::World.State.setPause(true);
 	}
-});
\ No newline at end of file
+
+	local onInit = o.onInit;
+	o.onInit = function()
+	{
+		onInit();
+		// Quick way to reduce the visibility of all unique locations by 30%. This does not affect the tundra elk location because it will be set to 0.8 at a later point
+		if (this.isLocationType(::Const.World.LocationType.Unique)) this.m.VisibilityMult *= 0.7;
+	}
+});
```

Comment:
Why are you doing this?

Reply:
I don't want legendary locations to be easier to find so this nerf should go alongside the buff from the new rumors.

Combing the whole world should not be the main strategy for finding all legendary locations. 
You will still be able to find them by accident but that's now probably twice as hard with this nerf.

Instead if is now roughly split between you finding half the locations by accident and the other half by following clues from rumors. And especially the last legendary locations will be much more deterministic to find because all legendary rumors will point to them.

Reply:
Now revisiting this my method here may be flawed.
I assumed that the onInit function is only ever called once for every location ever: right after the `create()`

But if onInit is called every time that they are loaded from a savegame then this will continuously decrease their visionMult every time you save and load

Reply:
I removed this vision penalty now for this PR.